importer/argument_parser.py

```diff
@@ -35,6 +35,8 @@ def _build_preparation_parser(self, preparation_parser):
         preparation_parser.add_argument('-t', '--ticket', type=int, required=True, help='RT Ticket number')
         preparation_parser.add_argument('-i', '--input', required=True, help='Directory containing the read files')
         preparation_parser.add_argument('-o', '--output', required=True, help='Base directory for import datas')
+        preparation_parser.add_argument('-b', '--breakpoint', type=int, required=False, default=150,
```

Comment:
I couldn't see the corresponding changes to the load functionality. If they are not there, I would keep the default to either a very large int or to 0, assuming that 0 disable this.